src/cmds/orgs:create.ts

```diff
@@ -26,6 +26,26 @@ export const builder = {
   },
 };
 
+export async function createOrg(
+  file: string,
+  includeExistingOrgsInOutput: boolean,
+  noDuplicateNames?: boolean,
+): Promise<string> {
+  const res = await createOrgs(file, {
+    noDuplicateNames,
+    includeExistingOrgsInOutput,
+  });
+
+  const orgsMessage =
+    res.orgs.length > 0
+      ? `Created ${res.orgs.length} out of ${res.totalOrgs} organization(s). Written the data to file: ${res.fileName}`
+      : `⚠ No organization(s) created!`;
+
+  console.log(orgsMessage);
+
+  return res.fileName;
```

Comment:
can this be an object `{ filename: res.fileName}` then it is easy for us to extend and return more if we need to